apps/i18n/studio/en_us.json

```diff
@@ -2,6 +2,12 @@
   "actor": "actor",
   "addCharacter": "add a",
   "addCharacterTooltip": "Add a character to the scene.",
+  "addPoints10": "add 10 points",
```

Comment:
and, yes, we typically use combo boxes in blockly. we do have `params` version of some blocks in playlab, but they make the student work a bit more complex. This seemed like a good case to keep it simple. We can always add another `params` version of this block later.